eng/pipelines/freebsd.yml

```diff
@@ -0,0 +1,27 @@
+# FreeBSD leg. Only for used in Official Builds
+
+jobs:
+- template: corefx-base.yml
+  parameters:
+    isOfficialBuild: true
+    targetOS: FreeBSD
+    jobs:
+
+    - job: FreeBSD
+      strategy:
+        matrix:
+          x64_Release:
+            _configuration: Release
+            _architecture: x64
+            _framework: netcoreapp
+            _buildScriptPrefix: 'export DotNetCoreSdkDir=/usr/local/dotnet/ && export DotNetRoot=/usr/local/dotnet/ && '
```

Comment:
Do we need the export here or would "foo=bar ./build.sh ..." work as well?

Reply:
I didn’t try that. I will try it cause I like that syntax better. I just copied over from the old definition.